.gitignore

```diff
@@ -4,3 +4,131 @@ bun.lockb
 .idea
 .log*
 .vscode
+/.npm/
```

Comment:
this all is suitable for a global .gitignore, not for the repo one

Reply:
The last two commits were dependent on my environment, but I accidentally included them without changing branches. I will think of a countermeasure such as changing the branch.

Reply:
@shanginn Would you think which is good on this repository revert or force push to undo my last two commits?

Reply:
I mean it's your repo until it's merged. so it's your choice :)

I myself probably prefer to revert to the first commit and force push in this situation

Reply:
Thanks for sharing your thoughts, I will use force push.